doc/man3/X509_check_issued.pod

```diff
@@ -14,13 +14,13 @@ certificate
 
 =head1 DESCRIPTION
 
-This function checks if certificate I<subject> was issued using CA
-certificate I<issuer>. This function takes into account not only
-matching of issuer field of I<subject> with subject field of I<issuer>,
-but also compares B<authorityKeyIdentifier> extension of I<subject> with
-B<subjectKeyIdentifier> of I<issuer> if B<authorityKeyIdentifier>
-present in the I<subject> certificate and checks B<keyUsage> field of
```

Comment:
Dropping the keyusage check in existing public API function is an API break. Although the new function is much more logical perhaps the keyusage check should be dropped only in case the certificate is self-signed? That would IMO make the API break harmless.

Reply:
Good point.
Droping of the key usage check in this PR was just due to trying to keep this backport minimal.
I've pushed a fixup that now does more like the master PR #10587, introducing a further internal function that differentiates tasks better, in a way that the original key usage check is maintained in the exported function `X509_check_issued()`.